src/trace-agent.js

```diff
@@ -35,8 +36,10 @@ var traceAgent;
 function TraceAgent(config, logger) {
   this.config_ = config;
   this.logger = logger;
+  this.plugins = config && config.plugins ? config.plugins : {};
```

Comment:
@DominicKramer I think we should not fail if `plugins` aren't defined. We don't want to prohibit custom trace spans in the absence of tracing well-known libraries.